vars/Debian.yml

```diff
@@ -19,3 +19,6 @@ __patroni_system_packages:
   - { name: "python3-psycopg2", state: "present" }
   - { name: "python3-pip",      state: "present" }
   - { name: "jq",              state: "present" }
+
+__patroni_packages:

```

Comment:
We already have the `__patroni_system_packages` variable to install packages via yum/apt. I would like to avoid having multiple variables for the same purpose.
IMHO it's okay to override `__patroni_system_packages` manually if `patroni_install_from_pip` is set to `true`. What do you think?